_assets/js/netlify/topics/preview.js

```diff
@@ -0,0 +1,34 @@
+import makeBodyContent from "./makeBodyContent";
+import makeTemplate from "./makeTemplate";
+
+let preview = createClass({
+    render: function () {
+      const entry = this.props.entry;
+      const title = entry.getIn(['data', 'title']);
+      const leadText = entry.getIn(['data', 'lead']);
+      const rawBodyContent = entry.getIn(['data', 'body']);
+      const bodyContentsArr = makeBodyContent(rawBodyContent);
+      const publishDate = entry.getIn(['data', 'publish-date']);
+      const updatedDate = entry.getIn(['data', 'updated-date']);
+      const print = entry.getIn(['data', 'print']);
+      const sideNavPDF = entry.getIn(['data', 'sidenav-pdf']);
+      const relatedContent = entry.getIn(['data', 'related-content']);
+      const pageData = {
+        title: title,
+        leadText: leadText,
+        body: bodyContentsArr,
+        publishDate: publishDate,
+        updatedDate: updatedDate,
+        print: print,
+        pdf: sideNavPDF,
+        relatedContent: relatedContent,
+      }
+      return makeTemplate(pageData);
+    },
+  });
+
+  CMS.registerPreviewTemplate('topics', preview);
+  CMS.registerPreviewTemplate('resources', preview);
+  CMS.registerPreviewTemplate('resources', preview);
+  CMS.registerPreviewTemplate('law-and-regs', preview);
+  CMS.registerPreviewTemplate('pages', preview);
```

Comment:
```suggestion
  const pages = [
    'topics',
    'resources',
    'resources',
    'law-and-regs',
    'pages'
  ];
  pages.forEach(page => CMS.registerPreviewTemplate(page, preview));
```

happy either way with this:
  - splitting it to an array will help you more easily scan to see the duplicate `'resources'`.
  - Keep it as is will help if there's pages where you need to specify something other than `page, preview`